src/ign.cc

```diff
@@ -523,10 +540,157 @@ extern "C" IGNITION_FUEL_TOOLS_VISIBLE int downloadUrl(const char *_url,
       return false;
     }
   }
+  // Collection?
+  else if (client.ParseCollectionUrl(url, collection))
+  {
+    if (ignition::common::Console::Verbosity() >= 3)
+    {
+      std::cout << "Downloading collection: " << "\033[36m" << std::endl
+                << collection.AsPrettyString("  ") << "\033[39m" << std::endl;
+    }
+
+    bool downloadModels = true;
+    bool downloadWorlds = true;
+    if (nullptr !=_type)
+    {
+      if (strcmp(_type, "model") == 0)
+      {
+        downloadWorlds = false;
+      }
+      else if (strcmp(_type, "world") == 0)
+      {
+        downloadModels = false;
+      }
```

Comment:
Should you handle the `else` case as an error?

Reply:
Shouldn't be reachable from the `ign` tool because an error is thrown in the ruby script, but I added an else in case the function is used elsewhere. https://github.com/ignitionrobotics/ign-fuel-tools/pull/98/commits/d18c3b870b2e3f58d01ac29fbd71eead44386741